src/wp-includes/theme.php

```diff
@@ -959,7 +959,20 @@ function validate_theme_requirements( $stylesheet ) {
 		);
 	}
 
-	return true;
+	/**
+	 * Filters the theme requirement validation response.
+	 *
+	 * If a theme fails due to a Core-provided validation (incompatible WP, PHP versions), this
+	 * filter will not fire. A WP_Error response will already be returned.
+	 *
+	 * This filter is intended to add additional validation steps by site administrators.
+	 *
+	 * @since 6.0.0
```

Comment:
The filter introduces for future versions so it should be **@since 5.9**

Reply:
Happy to change. I'm not presuming it'll land in time for 5.9 though.